amt/site/templates/layouts/base.html.j2.webpack

```diff
@@ -17,19 +17,23 @@
     <% } %>
     <script type="text/javascript" src="{{ url_for_cache('static', path='vendor/htmx/js/json-enc.js') }}"></script>
     <script type="text/javascript" src="{{ url_for_cache('static', path='vendor/htmx/js/response-targets.js') }}"></script>
-
+    <script type="text/javascript" src="https://unpkg.com/@nl-rvo/design-tokens/dist/index.js"></script>
```

Comment:
I would prefer using NPM because you can version it and you have more control. now the application can break when someone changes/deletes something from unpkg.

Reply:
Yeah agree, I and Robbert will change this to use the NPM packages :)